.github/workflows/ci-test.yml

```diff
@@ -206,7 +206,7 @@ jobs:
 
     - name: Build (MacOS)
       run: |
-          ./bootstrap.sh -DENABLE_GC=ON -DCMAKE_BUILD_TYPE=RELEASE -DENABLE_UNIFIED_COMPILATION=ON && cd build && make -j2
+          ./bootstrap.sh -DENABLE_GC=ON -DCMAKE_BUILD_TYPE=RELEASE -DENABLE_UNIFIED_COMPILATION=ON -DProtobuf_LIBRARIES=/usr/local/Cellar/protobuf/21.12/lib/ -DProtobuf_INCLUDE_DIR=/usr/local/Cellar/protobuf/21.12/include/ && cd build && make -j2
```

Comment:
This will likely fail the next time protobuf is updated in `brew`, there should be some way to get the paths from `brew`, or maybe from the `protoc` binary. (I'm not macOS user, cannot test it).